package.json

```diff
@@ -7,6 +7,7 @@
     "url": "git@github.com:raml-org/raml-js-parser.git"
   },
   "main": "./lib/raml.js",
+  "bin": "bin/raml-parser.js",
```

Comment:
Usually binaries come w/o extension otherwise it'll look weird to run `raml-parser.js ...`. Imaging you'd run grunt or gulp as `grunt.js ...` or `gulp.js ...` :)